### DISCUSSION TITLE
Plugin icons not showing

### DISCUSSION THREAD

**Discussion by MiltonBalaOfficial** [xournalpp/xournalpp#6036] [Q&A]
Hi, I was using `xournalpp-1.2.3+dev-nightly.20241019-windows` in my windows 11 PC. The plugin icons were showing properly but today I install `xournalpp-1.2.4+dev-nightly.20241107-windows` and the icons are not showing properly. Just folder-like icons are showing for all the plugins. The icons are placed in `C:\Users\<user>\AppData\Local\share\icons\hicolor\scalable\actions`. What should I do now for the proper viewing of the icons?

**Comment by rolandlo** [comment]
Somehow the icons are not found and the `image-missing` icon is used as a replacement. I have no idea why it happens to you though. I can't reproduce on Linux.

**Reply by MiltonBalaOfficial** [reply]
I reinstall `xournalpp-1.2.3+dev-nightly.20241019-windows` and the icons are showing properly and then again the `xournalpp-1.2.4+dev-nightly.20241107-windows` but same result!

**Reply by rolandlo** [reply]
I will check on Windows once I get to it.

**Reply by MiltonBalaOfficial** [reply]
I uninstall xournal and then install `1.2.4+dev-nightly.20241107-windows` and the problem gone
